client/clientauth.js

```diff
@@ -61,7 +62,7 @@ module.exports = ( fastify, opts, done ) =>
 				}
 				catch
 				{
-					return { success: false, error: STRYDER_RESPONSE }
+					return { success: false, error: STRYDER_RESPONSE, response: authResponse.toString() }
```

Comment:
Going through this again, did you (get the opportunity to) check what is contained in `authResponse`? Just wanna avoid the situation of someone sending a screenshot containing `authResponse` content into #general when it actually contains some form of auth token. ^^

Like something along the lines of `Failed authenticating with token ABCXYZ`.

Reply:
Normal launcher doesn't ever log the actual response anyway, only those running my launcher PR will, which for now, hasn't been merged. I don't really have a way to test it, since I can't reproduce locally. However with that being said, if it turns out that it does print tokens using my launcher PR, I will update that PR to avoid that case before it gets merged

Reply:
In that case it's fine by me ^^